app/forms/candidate_interface/other_qualification_details_form.rb

```diff
@@ -13,7 +13,7 @@ class OtherQualificationDetailsForm
 
     validates :qualification_type, presence: true
 
-    validates :award_year, presence: true
+    validates :award_year, presence: true, numericality: { only_integer: true }
```

Comment:
Can the numericality validator enforce the range we want to validate against?
https://guides.rubyonrails.org/active_record_validations.html#numericality ie. with the `:in` option?
It would save having 2 validations here.

Reply:
Thanks, I've added